analyses/molecular-subtyping-ATRT/02-ATRT-molecular-subtyping-plotting.R

```diff
@@ -12,20 +12,6 @@
 
 #### Set Up --------------------------------------------------------------------
 
-# Install and load in ComplexHeatMap
-if (!("ComplexHeatmap" %in% installed.packages())) {
-  install.packages("ComplexHeatmap")
-}
-library(ComplexHeatmap)
```

Comment:
I accidentally deleted this so I ended up adding in the necessary `ComplexHeatmap::` bits.